src/test/java/com/emc/mongoose/storage/driver/kafka/integration/KafkaTopicTest.java

```diff
@@ -0,0 +1,64 @@
+package com.emc.mongoose.storage.driver.kafka.integration;
+
+import static org.junit.Assert.assertTrue;
+
+import com.emc.mongoose.storage.driver.kafka.util.docker.KafkaNodeContainer;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Set;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.junit.*;
+
+public class KafkaTopicTest {
+  private static KafkaNodeContainer kafkaNodeContainer;
+
+  private AdminClient adminClient;
+
+  @BeforeClass
+  public static void initClass() throws Exception {
+    kafkaNodeContainer = new KafkaNodeContainer();
+  }
+
+  @AfterClass
+  public static void shutDownClass() {
+    kafkaNodeContainer.close();
+  }
+
+  @Before
+  public void initTest() {
+    String ip = kafkaNodeContainer.getKafkaIp();
+    adminClient =
+        AdminClient.create(
+            Collections.singletonMap(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, ip));
+  }
+
+  @After
+  public void shutDownTest() {
+    adminClient.close();
+  }
+
+  @Test
+  public void createSingleTopicTest() throws Exception {
+    adminClient
+        .createTopics(Collections.singleton(new NewTopic("test-topic", 1, (short) 1)))
```

Comment:
please use some test local topic name, e.g. "topic" + KafkaTopicTest.class.getSimpleName()
different tests may use same kafka instance in future
so we need to avoid the collision